Formula/mediaconch.rb

```diff
@@ -36,7 +36,7 @@ def install
               "--disable-dependency-tracking",
               "--enable-static",
               "--enable-shared",
-              "--with-libcurl",
+              "--with-libcurl=runtime",
```

Comment:
MediaInfoLib now rely on pkg-config libcurl.pc file for dynamic linking.

I'm using dlload instead, to avoid extra non needed dependency (macOS already provide libcurl)